### ISSUE TITLE
BMC crashes after starting another server

### ISSUE THREAD

**Issue by MyNameIsTobi** [LunaPixelStudios/Better-MC#1952]
### Describe the Bug

Im running a BMC4 Server. After i installed a new one, in antoher directory, the old one stops immediately. If i restart that crashed server, the new installed crashes immediately.

### Reproduction Steps

1. start one server
2. start another server
3. first server crashes

### Crash Report or Log

Terminal Output: https://pastebin.com/vQzeG1nB - Complete Log: https://pastebin.com/Cvssi54M

### Operating System

Ubuntu 24.04.1 LTS

### Minecraft Version

1.20.1

### Modloader

Forge

### Modpack Version

v34.5 (old), v35 (new)

**Comment by Frontear** [comment]
First time we've ever seen this issue. Could you please try using a more updated version of Java, so we can determine if its a JVM bug or our mod.

Could you also clarify your system's setup. Is this a VM? Are you using a special hardened kernel? Any customized kernel module configurations? Basically, as much information as you can provide. The crash seems to indicate a failure whilst the network library executed a timeout handler, which is surprising since we've never seen this issue before. Knowing more could be helpful in isolating the issue.

**Comment by MyNameIsTobi** [comment]
It's a dedicated server hosted by hetzner, no VM. 
Hardware is: Ryzen 7 7700, 64GB Ram and 2 NVMe disks.
Ubuntu was installed like 3 weeks ago. No special kernals etc. 
Installed software besides Minecraft is Apache, MySQL and PHP.

I now updated to Java Version 21. All system updates are also done. 

I tried another instance of Minecraft with Fabric, this is running fine. 
I also installed the newest version of the modpack with nothing configured but the ports and right after 'Preparing level "world"' the other running instance crashes.

**Comment by Frontear** [comment]
Could you try and remove the Luna mod from the mods folder? I'm still not sure _why_ this issue happened, but since it's directly related to that mod removing it should hopefully bypass it.

**Comment by MyNameIsTobi** [comment]
Yes, this now works perfectly fine. I can run as much instances as i want. Thank you so much. You know exactly what the problem is?

**Comment by Frontear** [comment]
Unfortunately no. I'm not able to reproduce this issue in the slightest. It's also supposedly occuring in the netty.io library, which means its well out of our hands (Luna is blamed since it uses said library).

It could be hardware related or software related, maybe its a bug in the library itself. You won't miss anything by removing Luna do, it doesn't do anything except provide a free world sharing service. Removing it will not affect the modpack in any way.
